.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: minimal
+language: cpp
```

Comment:
@PiotrSikora this was done on purpose. For details on why this was needed look at https://github.com/lyft/envoy/pull/882.